src/localisation/events.ts

```diff
@@ -0,0 +1,7 @@
+export const addButtonLabel = "Add New Event";
+
+export const editButtonLabel = "Edit Event Details";
+
+export const deleteButtonLabel = "Delete";
+
+
```

Comment:
This implementation was just a draft on other places. We are directly using text on components. So i don't think we need to continue using it. Was my fault though to not remove.